src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,13 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
-        return null;
+    public boolean findByEmail(String email, String password) {
+        boolean check = false;
+        for (int i = 0; i < users.length;i++) {
+            if ((users[i].getEmail().equals(email)) && (users[i].getPassword().equals(password))) {
+                check = true;
+            }
```

Comment:
use for each instead of for i, remove redundant parentheses, pay attention to spaces and codestyle